src/cryptography/hazmat/primitives/twofactor/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import unicode_literals
+
+import base64
+
+from six.moves.urllib.parse import quote, urlencode
+
+
+__all__ = ['get_provisioning_uri']
+
+
+def get_provisioning_uri(otp, account_name, issuer=None, counter=None):
+    """Generates a provisioning URI which can be recognized by Two-Factor
```

Comment:
Assuming we decide to pull this in we put documentation separately in restructured text. You'll need to add this to the docs there.